packages/components/src/components/combobox/shadow.tsx

```diff
@@ -279,6 +284,11 @@ export class KolCombobox implements ComboboxAPI {
 				});
 				break;
 			}
+			case 'Tab': {
```

Comment:
Für welchen Fall brauchen wir das? Da wir das `onChange`-Event des InputText schon verarbeiten, führt es in meinen Tests zu einem doppelten change-Event, wenn man Tab drückt.

Ich sehe die Lösung auch sonst eher kritisch, weil es zwar den <Tab>-Fall berücksichtigt, aber keine der anderen Methoden, wie die Komponente den Fokus verlieren kann.